roles/setup_replication/tasks/rm_replication.yml

```diff
@@ -15,9 +16,10 @@
   ansible.builtin.systemd:
     name: "{{ pg_service }}"
     state: stopped
+    enabled: false
   when:
-    - ansible_facts.services[pg_service] is defined
-    - ansible_facts.services[pg_service].state == 'running'
+    - ansible_facts.services[pg_service + '.service'] is defined
```

Comment:
Do you think we should append the suffix '.service' to the service name specified in the variable, and revise the documentation to explain to users how to utilize the service name variable?

Reply:
The only place this suffix is necessary is when calling the `ansible_facts.services`, while the `ansible.builtin.systemd` only needs the currently defined `pg_service`. We also call the `pg_service` variable to build other service names so adding the `.service` suffix may require us to remove it in other places.